procedures/accounts_setup.rst

```diff
@@ -0,0 +1,19 @@
+Here is a list of accounts and access needed to be successful in Release Engineering
+
+General:
+ * `JIRA <https://jira.mozilla.com/projects/RELENG/issues>`_ - add username to Mozilla instance
+ * `Commit Access <https://www.mozilla.org/en-US/about/governance/policies/commit/access-policy/>`_ - fill out form, file bug
+ * Bugzilla
+ * Phabricator
+ * Github
+ * `Mozilla VPN <https://mana.mozilla.org/wiki/pages/viewpage.action?pageId=30769829>`_
+
+Release Duty:
+ * ?
```

Comment:
https://moz-releng-docs.readthedocs.io/en/latest/procedures/release-duty/index.html#communication and https://moz-releng-docs.readthedocs.io/en/latest/procedures/release-duty/index.html#permissions already define what's needed, maybe we should just put the two references in here?